src/domain/certificates/useCases/getChain.js

```diff
@@ -1,6 +1,42 @@
 import { BLOCKCHAINS } from '../../../constants/blockchains';
 import addresses from '../../addresses';
 import { getText } from '../../i18n/useCases';
+import { capitalize } from '../../../helpers/string';
+
+
+// merkleRoot2019: see https://w3c-dvcg.github.io/lds-merkle-proof-2019/#blockchain-keymap
+function getMerkleRoot2019Chain (anchor) {
+  const supportedChainsMap = {
+    'btc': {
+      chainName: BLOCKCHAINS.bitcoin.name
+    },
+    'eth': {
+      chainName: BLOCKCHAINS.ethmain.name
+    }
+  };
+  const dataArray = anchor.split(':');
+  const chainIndex = dataArray.findIndex(data => Object.keys(supportedChainsMap).indexOf(data) > -1);
+  if (chainIndex > -1) {
+    const chainCode = dataArray[chainIndex];
+    const network = dataArray[chainIndex + 1];
```

Comment:
So one case this might not work is with mocknet. 

Right now I added mocknet support to the python library: https://github.com/blockchain-certificates/lds-merkle-proof-2019-py/blob/master/lds_merkle_proof_2019/mappings.py#L13

`blink:mocknet:582733d7cef8035d87cecc9ebbe13b3a2f6cc52583fbcd2b9709f20a6b8b56b3`

Given there's not really a network to it. It's sort of unofficial support for mocknet since that's our made-up thing. Not sure that'll get put into the MerkleProof2019 standard at all. 

We may have to coordinate on whether or not this works for us and if we want to change the structure of mocknet. 

But also given this (https://github.com/w3c-dvcg/lds-merkle-proof-2019/issues/5) and the fact that blink isn't a fully even a drafted standard yet, there may be different mappings depending on different chains. 

In the end there would probably need to be a `blink` library for this.

Reply:
Right, we can do a hard check with `mocknet` (since we wouldn't be entering this condition) and decide what to do specifically for it.

Regarding different chains, it would be as easy as adding them here: https://github.com/blockchain-certificates/cert-verifier-js/pull/403/files/9dfc95ea643090037cf7db1a3db672bd60061252#diff-efe81634eb97dd9985e8267325b6fd1cR9, as support is needed. I see in your proposal though that it could come as full word (and not just `eth` `btc`). Nothing unsurmontable but do you have a clear idea if this would go through?

I also see in your proposal that the `OPTIONAL_BLOCK` might come before the `TRANSACTION` which could be a problem regarding to this PR. We might have to revise the way we retrieve the id.

Reply:
Good catch on the full word vs code. I haven't thought about that and was inconsistent in the proposal. I have no idea when it'll go through, Manu was supposed to create it a few weeks ago I guess. 

I think for now it's safe to proceed unless something changes - targets can and will change throughout the verifiable credential ecosystem as time progresses anyways. 

I'll keep you up to date on any blink stuff but for now I think it should suffice to our use case.

Reply:
And yeah, if we hardcode in mocknet stuff, that may be okay.